web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -141,13 +143,20 @@
     <p class="text-lg text-immich-fg dark:text-immich-dark-fg">{$t('info')}</p>
   </div>
 
-  {#if asset.isOffline}
+  {#if isOffline}
     <section class="px-4 py-4">
       <div role="alert">
-        <div class="rounded-t bg-red-500 px-4 py-2 font-bold text-white">{$t('asset_offline')}</div>
+        <div class="rounded-t bg-red-500 px-4 py-2 font-bold text-white">
+          {$t('asset_offline')}
+        </div>
         <div class="rounded-b border border-t-0 border-red-400 bg-red-100 px-4 py-3 text-red-700">
           <p>
-            {$t('asset_offline_description')}
+            {#if $user?.isAdmin}
+              <p>{$t('admin.asset_offline_description')}</p>
+              <p class="text-sm">{$t('path')}: {asset.originalPath}</p>
```

Comment:
I think with regards to RTL languages this entire string needs to be localized (so passing the path as a variable), as the structure `Path: /path/to/asset` may not always work

Reply:
Fixed now